.github/pull_request_template.md

```diff
@@ -1,80 +1,35 @@
+This text is intended to be removed and replaced with actual description. Other items can also be removed or edited if not relevant. If any of following items require more description, it should also go in this block.
+Please, read all items and make an action every time to create more conscious PR. Thanks!
+
 ### PR fulfills these requirements
-- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
-- [ ] Tests for the changes have been added/updated (for bug fixes/features)
-- [ ] Docs have been added/updated (for bug fixes/features)
+- [ ] Tests for the changes have been added/updated
+- [ ] Docs have been added/updated
 - [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
-- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)
-
-
-
-#### Change has impacts in these area(s)
-_(check all that apply)_
-- [ ] Product design
-- [ ] Frontend
-
-
+- [ ] Self-reviewed and ran all changes on a local instance
 
 ### Describe the reason for change
-_(link to issue, supportive screenshots etc.)_
-
-
-
-#### What does this fix?
-_(if this is a bug fix)_
-
-
-
-#### What is the new behavior?
-_(if this is a breaking or feature change)_
-
-
-
-#### What is the current behavior?
-_(if this is a breaking or feature change)_
-
-
+(if it's not clear from description)
 
-#### What libraries were added/updated?
-_(list all with version changes)_
-
-
-
-#### Does this change affect performance?
-_(if so describe the impacts positive or negative)_
-
-
-
-#### Does this change affect security?
-_(if so describe the impacts positive or negative)_
-
-
-
-#### What alternative approaches were there?
-_(briefly list any if applicable)_
-
-
-
-#### What feature flags were used to cover this change?
-_(briefly list any if applicable)_
+### What feature flags were used to cover this change?
 
+### What alternative approaches were there?
 
+### This change affects (describe how if yes)
+- [ ] Performance
+- [ ] Security
+- [ ] UX
 
 ### Does this PR introduce a breaking change?
-_(check only one)_
 - [ ] Yes, and covered entirely by feature flag(s)
 - [ ] Yes, and covered partially by feature flag(s)
 - [ ] No
 - [ ] Not sure (briefly explain the situation below)
 
-
-
 ### What level of testing was included in the change?
-_(check all that apply)_
-- [ ] e2e
-- [ ] integration
-- [ ] unit
-
-
+- [ ] e2e (codecept)
+- [ ] integration (cypress)
+- [ ] unit (jest)
 
 ### Which logical domain(s) does this change affect?
-_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_
+(comma-separated list like "Taxonomy, Dynamic Children")
```

Comment:
This one as well I think in the context of OSS, but also our own ability to use this information, would it not be better to just allow for the **Reason for change** section to facilitate this? As part of the reason for change, I often find myself writing in a lot of these other sections information, and in general I don't see a lot of people filling it in. I think if we collapse this all the way down, we can then use **Reason for change** as the litmus test of the reviewers ability to understand what they are reviewing. If the change is not obvious after seeing the excerpt describing the change, or it doesn't align well with the code, then it is a single point to call out to get alignment and clarity which helps cut down on the amount of QA reopens.

Reply:
Huge plus on ability to use this info! This section is really no use for anything, thanks.
About the rest — yes, I'm also fan of good descriptions with all related info there, and I even tried to make such description placeholder, that it will push people towards more thoughts and nuances.

Reply:
For me, it helps to systematize what has been done. And also shows whether the changes were intended. It can be especially useful if debugging an existing problem leads us to the specific PR.

In general, it is not about the cause, but about what has been affected. It's the same as tags for feature-flags. If tags are good you can realize which FF can cause the problem you have found.

Reply:
About FFs — yes, because there is a search by tags. But how this list will help us and when will it be not deferred from name and description?